main.py

```diff
@@ -0,0 +1,49 @@
+import random
+
+def input_name():
+    """
+    Asks the user to input a list of names, validates them, removes duplicates and returns a list of valid names.
```

Comment:
Perhaps you could add slightly more detail to your functions. Ie what sort of data type does this function accept and return? Could be useful for larger projects